bundles/AdminBundle/Resources/views/Admin/Login/lostpassword.html.twig

```diff
@@ -3,11 +3,19 @@
 {% block content %}
 
     {% if app.request.method == 'POST' %}
-        <div class="text error">
-            {{ 'A temporary login link has been sent to your email address.'|trans([],'admin') }}
-            <br/>
-            {{ 'Please check your mailbox.'|trans([],'admin') }}
-        </div>
+        {% if error is null %}
+            <div class="text success">
+                {{ 'A temporary login link has been sent to your email address.'|trans([],'admin') }}
+                <br/>
+                {{ 'Please check your mailbox.'|trans([],'admin') }}
+            </div>
+        {% else %}
+            <div class="text error">
+                {{ 'An error occured when resetting password'|trans([],'admin') }}
+                <br/>
```

Comment:
```suggestion
                {{ 'An error occured when resetting password:'|trans([],'admin') }}
                <b>{{ error|trans([],'admin') }}</b>
                <br/>
```
I think it is better to provide the error message as well to help in debugging the cause.